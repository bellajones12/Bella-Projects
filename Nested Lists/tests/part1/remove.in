NEW 0
NEW 1
HI
REMOVE 1
