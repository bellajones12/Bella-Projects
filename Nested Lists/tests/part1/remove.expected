List 0: 
List 1: HI
List 1 has been removed.

Number of lists: 1
List 0
